### ISSUE TITLE
Issue with synthetic\roijers_train.py

### ISSUE THREAD

**Issue by ghost** [RunzheYang/MORL#22]
Hello,
   
    In your file: roijers_train.py, you have:
       
  agent.memorize(state, action, next_state, reward, terminal, roi=True)
  loss += agent.learn(corner_w)
But your code in the agent does not take input roi in memorize and corner_w in learn. It can only be run like this:
  loss += agent.learn()
  agent.memorize(state, action, next_state, reward, terminal)

   Thanks!
